src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -220,15 +220,15 @@ public static ProfessionalUser mapUserInfo(ProfessionalUser user, ResponseEntity
 
     public static String removeEmptySpaces(String value) {
         String modValue = value;
-        if (!StringUtils.isEmpty(modValue)) {
+        if (!ObjectUtils.isEmpty(modValue)) {

```

Comment:
StringUtils.isNotBlank may be good to use this

Reply:
As per Shreedhar's comment, `ObjectUtils.isEmpty()` is the replacement for the deprecated method:
https://github.com/hmcts/rd-professional-api/pull/1038#discussion_r625824967

Reply:
! is bit weird and not readable hence I feel use isNotBlank . is it isNotBlank deprecated?

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1038/commits/831f0951dda386d3bb8ee6124ead2b24550ff875